ui/static/ui/js/listing/listing.jsx

```diff
@@ -158,7 +161,9 @@ define('listing',
           // Keep track of resource id so we can lazy load panels.
           currentResourceId: undefined,
           // What panels are already loaded
-          loadedPanels: {}
+          loadedPanels: {},
+          isLearningResourcePanelDirty: false,
+          requestedPanelClose: false
```

Comment:
If you handle the close logic inside `this.closeLearningResourcePanel` you shouldn't need this piece of state anymore